FSQLocationBroker/FSQLocationBroker.h

```diff
@@ -402,8 +392,6 @@ typedef NS_OPTIONS(NSUInteger, FSQLocationSubscriberOptions) {
 
```

Comment:
Might need `nullable` above here on the error parameter in line 391, and might need a generic type on the array in line 377

Reply:
I don't think these need to be `nullable` either since again the`CLLocationManagerDelegate` does not have these as nullable params either.

Reply:
I think we guarantee an NSError object when `locationManagerFailedWithError:` is called